e2etests/testcase_test.go

```diff
@@ -3887,31 +3899,6 @@ Applications using RegexRequestMatcher with '.' in the regular expression are po
 				AddedBy:       "sha256:3fa3f612bdcb92746bf76be1b9c9e1c1c80de777aedaf48b7068f4a129ded3c2",
 				FixedBy:       "4.10.1685679861-1.el8",
 				Vulnerabilities: []apiV1.Vulnerability{
-					{
```

Comment:
You know I have seen this flap sometimes, and I have yet to dig into why this happens. Any ideas? I recall this was a key vulnerability we wanted to track because it's an unfixed OpenShift-related vulnerability which we did not track for some time. It'd be nice to keep this, but since we are focused more on Scanner V4, I'm ok with commenting this out with a ticket to go with it to track it

Reply:
Hrm, interesting, I'll see if I have an old copy of the data for comparing to see what changed - I did check and none of the RHSA's listed in the current scan results for this image include this particular CVE. 

I did notice CI uses a new 'genesis dump' each run - and if I remember correctly scanner currently does not handle 'deleted' vulns when processing a diff. So this would only appear in scanners that include a new genesis dump as of whatever date when this vuln was seemingly removed from the feeds.

Reply:
Resolving this - you were right the results flapped and were back in the next run.